changelog/pending/20240710--sdk-go--add-support-for-invoke-transforms.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: sdk/go
+  description: Add support for invoke transforms
```

Comment:
Nit: perhaps we call this out as top-level/global transforms (and in the PR title/commit message), if that's where we're starting?

Reply:
Makes sense.  Maybe by calling it "stack invoke transforms"?  Or did you have something else in mind?